src/ws/cockpitauth.c

```diff
@@ -1116,7 +1118,12 @@ cockpit_session_launch (CockpitAuth *self,
            g_str_equal (type, "tls-cert"))
     {
       if (command == NULL && unix_path == NULL)
-        command = cockpit_ws_session_program;
+        {
+          if (cockpit_ws_session_program)
```

Comment:
You removed the fallback code path but the commit message still claims "Fall back to calling cockpit-session directly for custom setups."

Reply:
Fixed the message, thanks!